apkid/rules/apk/protectors.yara

```diff
@@ -88,3 +88,25 @@ rule vkey_apk : protector
     1 of ($asseta*) and
     1 of ($assetb*)
 }
+
+rule kiwisec : protector
+{
+  meta:
+    description = "KiwiSec ApkProtector"
+    url         = "https://github.com/iKiwiSec/KiwiApkProtect"
+    author      = "Govind Sharma"
+    sample      = "d108652bd1b685765e3ada2b7376e3c3ff67f8162afcf8bad91e0aef79b7b08a"
+
+  strings:
+     $kw1  = "libkwscmm.so"
+     $kw2  = "libkwscr.so"
+     $kw3  = "libkwslinker.so"
+     $kw4  = "libKwProtectSDK.so"
+     $kw5  = "libKwAppGuardSDK.so"
+     $kw6  = "kwmkadp_arm64-v8a"
+     $kw7  = "kwmkadp_armeabi-v7a"
+     $kw8  = "kiwiguard.lic"
+
+  condition:
+    is_apk and any of ($kw*)
```

Comment:
Can this be 2 of without loss of accuracy?

Reply:
Found a sample (https://www.vmos.com/) that it might help you to clean up the rule:

```sh
   147168  1979-11-29 17:00   lib/armeabi-v7a/libvmtools.so
    83976  2022-02-19 09:58   lib/armeabi-v7a/libkiwi_dumper.so
    55132  2022-02-19 09:58   lib/armeabi-v7a/libkiwicrash.so
   130128  2022-02-19 09:58   lib/arm64-v8a/libkiwi_dumper.so
    80304  2022-02-19 09:58   lib/arm64-v8a/libkiwicrash.so
  9368512  2022-02-19 10:02   lib/arm64-v8a/libKwProtectSDK.so
  1705820  2022-02-19 10:04   lib/armeabi-v7a/libKwProtectSDK.so
```